package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "wormhole-solidity-sdk",
+  "version": "0.9.0",
+  "description": "Wormhole Solidity SDK",
+  "scripts": {
+    "test": "forge test",
+    "format": "forge fmt",
+    "prepublishOnly": "echo 'Error: Please run 'scripts/publish.sh' to publish.' && exit 1"
```

Comment:
Why error here instead of running the script? Presumably because the script is modifying the package.json?

I think the package.json should probably not have any scripts and instead publish should be implemented as a target in the Makefile.

Reply:
This is the same method used by OpenZeppelin and major libraries developers to be sure that you can import files without specifying `src` or `contracts`. In other words, the `package.json` file that is used to publish is a modified version of the primary one, in order to specify the files we want to publish and exclude anything else.

I wrote a `publish.sh` script because it was simpler and less invasive. Of course, you guys can put that stuff in your building/deploying process, which I think would be better.

Reply:
BTW, if you do not block the prePublish, someone can accidentally publish it not in the best way, creating a potential disaster for devs who are using the package.